client/app/scripts/reducers/root.js

```diff
@@ -721,6 +724,34 @@ export function rootReducer(state = initialState, action) {
       return state.set('showingTroubleshootingMenu', !state.get('showingTroubleshootingMenu'));
     }
 
+    case ActionTypes.TOGGLE_CONTRAST_MODE: {
+      const modules = [
+        require.resolve('../../styles/main.scss'),
+        require.resolve('../../styles/contrast.scss')
+      ];
+      // Bust the webpack require cache to for a re-download of the stylesheets
+      modules.forEach((i) => {
+        const children = require.cache[i] ? require.cache[i].children : [];
+        children.forEach((c) => {
+          delete require.cache[c];
+        });
+        delete require.cache[i];
+      });
+
+      const contrastEnabled = state.get('contrastMode');
+      if (!contrastEnabled || action.enabled) {
```

Comment:
Shouldn't it be `&&` instead of `||` or am I missing something obvious here? :) Also, in line `746` I'd put a similar (symmetrical) condition - I assume the idea here is that `require.ensure` would be executed if and only if `contrastEnabled !== action.enabled`, right?